www/dusv03/homework-06/js/main.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * Long live Sparta! Vytvořte funkci, která vyřeší Caesarovu širfu. Funkce dostane 
+ * na vstup zašifrovaný text a také hodnotu, která byla použita při šifrování, a pak 
+ * vrátí dešifrovaný text. Předpokládejte pouze anglickou abecedu s velkými 
+ * písmeny, ostatní znaky ignorujte. Poté v konzoli dešifrujte/dešiftujte následující texty.
+ * 
+ * key used - encrypted text
+ *       19 - MPH MABGZL TKX BGYBGBMX: MAX NGBOXKLX TGW ANFTG LMNIBWBMR; TGW B'F GHM LNKX TUHNM MAX NGBOXKLX. - TEUXKM XBGLMXBG
+ *        5 - YMJWJ NX ST QFB JCHJUY YMJ QFB YMFY YMJWJ NX ST QFB. - OTMS FWHMNGFQI BMJJQJW
+ *       12 - M YMZ ITA PMDQE FA IMEFQ AZQ TAGD AR FUYQ TME ZAF PUEOAHQDQP FTQ HMXGQ AR XURQ. ― OTMDXQE PMDIUZ
+ * 
+ * Následně vytvořte uživatelské rozhraní, ve kterém bude možné zadat zmíněné dvě 
+ * vstupní hodnoty (zašifrovaný text a použitý klíč) a po kliknutí na tlačítko
+ * "Decipher!" se na určeném místě zobrazí dešifrovaný text. Rozhraní také vhodně
+ * nastylujte.
+ */
+
+var Deciphertext_part = document.querySelector('#to_decipher');
+var deci_index_part = document.querySelector('#deci_index');
+var output_part = document.querySelector('#output');
+var alphabet = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ';
+var shiftChar = function (c, shift) {
+    var position = alphabet.indexOf(c.toUpperCase());
+    if (position === -1) {
+        return c;
+    }
+    position -= shift;
+    if (position < 0) {
+        position += 26;
+
+    }
+    return alphabet.charAt(position);
+};
+
+var shiftString = function (str, shift) {
+    var arr = str.split("")
```

Comment:
nemusi se splitovat string, k jednotlivym pismenam lze dostat pres str.charAt(i)

Reply:
ale neim jak je tam vrátit jinak než přes to pole